### DISCUSSION TITLE
Assistance Windows Install

### DISCUSSION THREAD

**Discussion by Vilo23** [martomi/chiadog#239] [Q&A] [answered]
My apologies I am new to programming and I am just following instructions. I was going well on installing for Windows 10 but at this point I receive an error
This is fine
PS C:\Users\Myaccount\chiadog> py.exe -m venv venv
Then I type this
PS C:\Users\Myaccount\chiadog> ..\venv\Scripts\activate
(I get an error I dont know what exactly ..\venv\ part means but I get error)

.\venv\scripts\activate : The term '..\venv\scripts\activate' is not recognized as the name of a cmdlet, function,
script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
correct and try again.
At line:1 char:1
+ ..\venv\scripts\activate

**Comment by pieterhelsen** [comment, marked as answer]
There should be a space between `.` and `.\venv\scripts\activate`:
`. .\venv\script\activate`

**Reply by Vilo23** [reply]
Thanks for that ok so i did 
. (space) .\venv\scripts\activate

. : File C:\Users\myaccount\chiadog\venv\scripts\Activate.ps1 cannot be loaded because running scripts is disabled on
this system. For more information, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:3
+ . .\venv\scripts\activate
+   ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException

**Reply by Vilo23** [reply]
When I do this
PS C:\Users\myaccount\chiadog> py.exe -m venv venv

should the next line just come back as follows

PS C:\Users\myaccount\chiadog>

**Reply by Vilo23** [reply]
ok I just realised that worked but I have some policy related to security preventing the running of scripts.... hmmmm

**Reply by pieterhelsen** [reply]
Yeah, it's the beauty of Windows unfortunately... 

By default Windows doesn't allow you to run PowerShell scripts. 

https://dev.to/aka_anoop/enabling-virtualenv-in-windows-powershell-ka3
This allows you to set the ExecutionPolicy to Unrestricted, but do know that this lessens the security of your system, since it could allow _other_ (more malicious) scripts to run as well. It's a risk each of us has to decide for themselves whether it's worth taking :) For me it was... (though I switched to Linux not long after)

**Comment by Vilo23** [comment]
Thank you for your help its most appreciated
